userguide/kaui/includes/translation-files.adoc

```diff
@@ -0,0 +1,59 @@
+:icons: font
+:imagesdir: C:\_My Documents\FlowWritingLLC\Projects\Kill Bill\Documentation\killbill-docs\userguide\assets\img\kaui
+
+=== Overview
+
+Kill Bill supports the ability to generate emails and invoices in various languages as long as the required translation files are present in the system. Three tabs on the Tenant Configuration page enable you to upload these files:
+
+* Invoice Template
+* Invoice Translation
+* Catalog Translation
+
+image::TranslationFilesTabs.png[width=850,align="center"]
+
+The functionality of these tabs is very similar. To avoid repetition, we've included instructions  for just one tab, with any differences noted.
+
+==== Additional Translation Resources
+
+* https://docs.killbill.io/latest/invoice_templates.html[_Invoice Templates_]
+* https://docs.killbill.io/latest/email-notification-plugin.html[_Email Notification Plugin_]
+* https://docs.killbill.io/latest/internationalization.html[_Internationalization_]
+
+=== View Translation Tabs
+
+To get there:
+
+. Click on your username and tenant name in the upper right corner:
++
+image::Users-SelectTenantName.png[width=650,align="center"]
++
+Kaui displays the Tenant Configuration page.
++
+. Scroll down to see the translation tabs:
++
+image::TranslationFilesTabs.png[width=850,align="center"]
+
+=== Upload a Translation File
+
+. <<View Translation Tabs, Go to the Tenant Configuration page>> and scroll down to the translation tabs.
++
+. Select the appropriate translation tab for the file you want to upload.
++
+. Click the *Choose File* button, locate the translation file, and select it.
++
+Once you have selected the file, Kaui displays the filename next to the *Choose File* button.
++
+image::Translation-UploadInvTemplate.png[width=350,align="center"]
++
+. _For Invoice Templates only:_ If this invoice template supports the manual pay option (for accounts that have the `MANUAL_PAY` tag), select the *Manual Pay* checkbox. (For more information, see the https://killbill.github.io/slate/#invoice-template["Templates"] section of the API.)
++
+image::Translation-UploadInvTemplate-Manual.png[width=350,align="center"]
++
+. _For Invoice Translation and Catalog Translation only:_ In the *Locale* field, specify the language supported by the uploaded file using the ISO _country_language_ format.
++
+image::Translation-UploadInvTransFile.png[width=850,align="center"]
+
+. Click the *Upload* button.
++
+If the upload is successful, Kaui displays a success message at the top of the screen and lists the file on the tab.
+^^_I'm assuming this is what happens, but I received a Cloudflare error when I tried to upload._
```

Comment:
Attaching a screenshot in case you need one.
![invoice_template_upload](https://user-images.githubusercontent.com/85998496/165026468-7a0cefa8-ef17-476e-bcd6-7a811b5dba30.png)